Gemfile

```diff
@@ -28,3 +28,5 @@ group :coverage, optional: ENV['COVERAGE']!='yes' do
   gem 'simplecov-console', :require => false
   gem 'codecov', :require => false
 end
+
+gem 'rdoc' if RUBY_VERSION >= '3.1'
```

Comment:
https://github.com/voxpupuli/beaker/pull/1737 will make this obsolete, but I first want to release a minor version that adds Ruby 3.1 support, then do a major one.